src/docs/concepts-transactions/default-negotiation-process/index.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Default negotiation process
+slug: default-negotiation-process
+updated: 2025-09-16
+category: concepts-transaction-process
+ingress:
+  Sharetribe has a default negotiation transaction process that enables
+  users to negotiate a price for the transaction. This article explains
+  how the negotiation process works.
+published: true
+---
+
+# Negotiation process overview
+
+One of the default Sharetribe transaction processes includes price
+negotiation dynamics. The same process enables both forward and reverse
+transaction flows.
+
+- **Forward transaction flow**
+
+A forward transaction flow on a marketplace happens when the provider
+posts a listing, and customers can initiate a transaction against that
+listing by responding to the listing.
+
+- **Reverse transaction flow**
+
+A reverse transaction flow on a marketplace happens when the customer
+posts a listing, and providers can initiate a transaction against that
+listing by responding to the listing.
+
+<info>
+
+The Sharetribe Web Template supports a reverse transaction flow starting
+with version [XX.XX.XX](TODO). Forward transaction flow is not supported
+natively in the template at the moment.
+
+</info>
+
+In both forward and reverse flows, the provider provides the product or
+service in question, and the customer pays for the transaction.
+
+The _default-negotiation_ process does not include bookings or stock.
+You can customize the process to include availability handling or stock
+handling in addition to price negotiation. For bookings, we also offer
+an example transaction process
+[negotiated-booking](https://github.com/sharetribe/example-processes/blob/master/README.md#negotiated-booking)
+– it is not supported by default in the Sharetribe Web Template, so you
+will need to build a client-side implementation.
+
+The default-negotiation process has two loops: the
+[price negotiation loop](#price-negotiation-loop-in-default-negotiation)
+and the
+[change request loop](#change-request-loop-in-default-negotiation).
+
+With all transaction process loops, including the ones in this process,
+it is important to keep in mind that a single transaction can only have
+a maximum of 100 transitions. This means that any implementation you
+create must make sure that you restrict looping transitions after a
+certain threshold, because your users should always have enough
+transition quota left to finish the transaction
+
```

Comment:
This could be in <info> tags

Reply:
Agreed, good idea!

The template does handle the limitation by default. I've added a mention of that in the info box, and I'll link to the part where that's defined in Template. I'll also add a mention that the thresholds can be changed with custom dev.